static/compiler.js

```diff
@@ -1399,6 +1381,32 @@ Compiler.prototype.updateLibsDropdown = function () {
                 if (libsList.children().length > 0)
                     libsList.append($('<hr>').addClass('lib-separator'));
 
+                var onChecked = _.bind(function (e) {
+                    var elem = $(e.target);
+                    // Uncheck every lib checkbox with the same name if we're checking the target
+                    if (elem.prop('checked')) {
+                        var others = $.find('input[name=\'' + elem.prop('name') + '\']');
```

Comment:
This might be a bug on the original code. How are not the rest of the compiler libs unchecked by this `$.find`?